### PR TITLE
[ISSUE #4323] Rpc example add the default NamesrvAddr

### PR DESCRIPTION
**Pull request by li-xiao-shuang**
fix : #4323 

**Make sure set the target branch to `develop`**

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by hzh0425 — APPROVED** [review]

**Review by Oliverwqcwrw — APPROVED** [review]

**Review by tianliuliu — APPROVED** [review]

File: example/src/main/java/org/apache/rocketmq/example/rpc/AsyncRequestProducer.java [hunk 1]
```diff
@@ -28,36 +28,40 @@
...
+
     public static void main(String[] args) throws MQClientException, InterruptedException {
-        String producerGroup = "please_rename_unique_group_name";
-        String topic = "RequestTopic";
-        long ttl = 3000;
 
-        DefaultMQProducer producer = new DefaultMQProducer(producerGroup);
+        DefaultMQProducer producer = new DefaultMQProducer(PRODUCER_GROUP);
+        //You need to set namesrvAddr to the address of the local namesrv
+        producer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```
Comment (odbozhou) [inline example/src/main/java/org/apache/rocketmq/example/rpc/AsyncRequestProducer.java:41]: May cause the official website quick start to set namesrv through the command line to fail.

image

File: example/src/main/java/org/apache/rocketmq/example/rpc/RequestProducer.java [hunk 2]
```diff
@@ -23,23 +23,28 @@
...
+
     public static void main(String[] args) throws MQClientException, InterruptedException {
-        String producerGroup = "please_rename_unique_group_name";
-        String topic = "RequestTopic";
-        long ttl = 3000;
 
-        DefaultMQProducer producer = new DefaultMQProducer(producerGroup);
+        DefaultMQProducer producer = new DefaultMQProducer(PRODUCER_GROUP);
+        //You need to set namesrvAddr to the address of the local namesrv
+        producer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```
Comment (odbozhou) [inline example/src/main/java/org/apache/rocketmq/example/rpc/RequestProducer.java:36]: May cause the official website quick start to set namesrv through the command line to fail.

image

File: example/src/main/java/org/apache/rocketmq/example/rpc/ResponseConsumer.java [hunk 3]
```diff
@@ -33,46 +31,50 @@
...
     public static void main(String[] args) throws InterruptedException, MQClientException {
-        String producerGroup = "please_rename_unique_group_name";
-        String consumerGroup = "please_rename_unique_group_name";
-        String topic = "RequestTopic";
 
         // create a producer to send reply message
-        DefaultMQProducer replyProducer = new DefaultMQProducer(producerGroup);
+        DefaultMQProducer replyProducer = new DefaultMQProducer(PRODUCER_GROUP);
+        // You need to set namesrvAddr to the address of the local namesrv
+        replyProducer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```
Comment (odbozhou) [inline example/src/main/java/org/apache/rocketmq/example/rpc/ResponseConsumer.java:45]: May cause the official website quick start to set namesrv through the command line to fail.

image

File: example/src/main/java/org/apache/rocketmq/example/rpc/ResponseConsumer.java [hunk 4]
```diff
@@ -33,46 +31,50 @@
...
         replyProducer.start();
 
         // create consumer
-        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(consumerGroup);
+        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(CONSUMER_GROUP);
+        // You need to set namesrvAddr to the address of the local namesrv
+        consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
         consumer.setConsumeFromWhere(ConsumeFromWhere.CONSUME_FROM_LAST_OFFSET);
```
Comment (odbozhou) [inline example/src/main/java/org/apache/rocketmq/example/rpc/ResponseConsumer.java:52]: May cause the official website quick start to set namesrv through the command line to fail.

image

**Review by Kvicii — DISMISSED** [review]
LGTM

**Review by tsunghanjacktsai — APPROVED** [review]

**Pull request closed without merging**